app/stylesheets/global.scss

```diff
@@ -0,0 +1,12 @@
+@import 'config/timing';
+@import 'config/type';
+
+html,
+body {
+  font-family: 'Open Sans', sans-serif;
+}
+
+a {
+  @include link-transition( 600ms, ease);
```

Comment:
yeah good point, lets drop it for now. We can pull out the timings again later if we want.